com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeBrickIndex.cs

```diff
@@ -111,10 +136,18 @@ internal ProbeBrickIndex(ProbeVolumeTextureMemoryBudget memoryBudget)
             Profiler.EndSample();
         }
 
+        public int GetRemainingChunkCount()
+        {
+            return m_AvailableChunkCount;
+        }
+
         internal void UploadIndexData()
         {
-            m_PhysicalIndexBuffer.SetData(m_PhysicalIndexBufferData);
+            var count = m_UpdateMaxIndex - m_UpdateMinIndex + 1;
+            m_PhysicalIndexBuffer.SetData(m_PhysicalIndexBufferData, m_UpdateMinIndex, m_UpdateMinIndex, count);
             m_NeedUpdateIndexComputeBuffer = false;
+            m_UpdateMaxIndex = int.MinValue;
```

Comment:
This looks dangerous to me, we should set this to default values like this straight before we start computing the new valid indices or if not possible at least do some range check before actually setting data to avoid some weird numerical explosion (in count)

Reply:
I don't think we can reinit them just before computing new valid indices because this happens every time we add/remove bricks, so we would not be accumulating the right range of indices.
I can add range check before sending data though. Not sure if it should be an assert or a no-op+error message in case of error though... (in all cases, result will be broken)

Reply:
I think we should assert here as it means we fucked something up really bad :D